<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AnyMeal</string>
  </property>
  <property name="windowIcon">
   <iconset resource="anymeal.qrc">
    <normaloff>:/images/anymeal.png</normaloff>:/images/anymeal.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="reset_button">
        <property name="text">
         <string>&amp;Show all recipes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="title_label">
        <property name="text">
         <string>&amp;Title</string>
        </property>
        <property name="buddy">
         <cstring>title_edit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="title_edit">
        <property name="toolTip">
         <string>Search for title text.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="category_widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QRadioButton" name="with_category_radio">
           <property name="text">
            <string>w</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="without_category_radio">
           <property name="text">
            <string>w/o</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="category_label">
        <property name="text">
         <string>&amp;Category</string>
        </property>
        <property name="buddy">
         <cstring>category_edit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="category_edit">
        <property name="toolTip">
         <string>Search for categories.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="ingredient_widget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QRadioButton" name="with_ingredient_radio">
           <property name="toolTip">
            <string>Search for recipes with ingredient.</string>
           </property>
           <property name="text">
            <string>w</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="without_ingredient_radio">
           <property name="toolTip">
            <string>Search for recipes without ingredient.</string>
           </property>
           <property name="text">
            <string>w/o</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ingredient_label">
        <property name="text">
         <string>&amp;Ingredient</string>
        </property>
        <property name="buddy">
         <cstring>ingredient_edit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ingredient_edit">
        <property name="toolTip">
         <string>Specify the ingredient.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="filter_button">
        <property name="text">
         <string>&amp;Search</string>
        </property>
        <property name="icon">
         <iconset resource="anymeal.qrc">
          <normaloff>:/images/search.svg</normaloff>:/images/search.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QListView" name="titles_view">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="layoutMode">
        <enum>QListView::Batched</enum>
       </property>
       <property name="uniformItemSizes">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QTextBrowser" name="recipe_browser">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>3</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_new"/>
    <addaction name="action_import"/>
    <addaction name="action_export"/>
    <addaction name="action_preview"/>
    <addaction name="action_print"/>
    <addaction name="action_delete"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_open_converter"/>
    <addaction name="action_about"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="action_edit"/>
    <addaction name="action_add_to_category"/>
    <addaction name="action_remove_from_category"/>
    <addaction name="action_deduplicate"/>
    <addaction name="action_collect_garbage"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="tool_bar">
   <property name="windowTitle">
    <string>Tool Bar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_new"/>
   <addaction name="action_import"/>
   <addaction name="action_export"/>
   <addaction name="action_edit"/>
   <addaction name="action_add_to_category"/>
   <addaction name="action_remove_from_category"/>
   <addaction name="action_preview"/>
   <addaction name="action_print"/>
   <addaction name="action_open_converter"/>
   <addaction name="action_collect_garbage"/>
   <addaction name="action_deduplicate"/>
   <addaction name="action_delete"/>
   <addaction name="action_quit"/>
  </widget>
  <action name="action_quit">
   <property name="icon">
    <iconset resource="anymeal.qrc">
     <normaloff>:/images/quit.svg</normaloff>:/images/quit.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Exit application</string>
   </property>
   <property name="statusTip">
    <string>Exit application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_import">
   <property name="icon">
    <iconset resource="anymeal.qrc">
     <normaloff>:/images/import.svg</normaloff>:/images/import.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Import recipes</string>
   </property>
   <property name="toolTip">
    <string>Read MealMaster recipes from files</string>
   </property>
   <property name="statusTip">
    <string>Read MealMaster recipes from files</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="action_preview">
   <property name="icon">
    <iconset resource="anymeal.qrc">
     <normaloff>:/images/preview.svg</normaloff>:/images/preview.svg</iconset>
   </property>
   <property name="text">
    <string>Print p&amp;review</string>
   </property>
   <property name="toolTip">
    <string>Print preview of current recipe</string>
   </property>
   <property name="statusTip">
    <string>Print preview of current recipe</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_print">
   <property name="icon">
    <iconset resource="anymeal.qrc">
     <normaloff>:/images/print.svg</normaloff>:/images/print.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Print recipe</string>
   </property>
   <property name="toolTip">
    <string>Print current recipe</string>
   </property>
   <property name="statusTip">
    <string>Print current recipe</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="action_about">
   <property name="icon">
    <iconset resource="anymeal.qrc">
     <normaloff>:/images/anymeal.png</normaloff>:/images/anymeal.png</iconset>
   </property>
   <property name="text">
    <string>A&amp;bout AnyMeal</string>
   </property>
   <property name="statusTip">
    <string>About AnyMeal</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="action_delete">
   <property name="icon">
    <iconset resource="anymeal.qrc">
     <normaloff>:/images/delete.svg</normaloff>:/images/delete.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Delete recipes</string>
   </property>
   <property name="toolTip">
    <string>Delete selected recipes</string>
   </property>
   <property name="statusTip">
    <string>Delete selected recipes</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="action_export">
   <property name="icon">
    <iconset resource="anymeal.qrc">
     <normaloff>:/images/export.svg</normaloff>:/images/export.svg</iconset>
   </property>
   <property name="text">
    <string>Exp&amp;ort recipes</string>
   </property>
   <property name="toolTip">
    <string>Write MealMaster recipes to file</string>
   </property>
   <property name="statusTip">
    <string>Write MealMaster recipes to file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_edit">
   <property name="icon">
    <iconset resource="anymeal.qrc">
     <normaloff>:/images/edit.svg</normaloff>:/images/edit.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Edit Recipe</string>
   </property>
   <property name="toolTip">
    <string>Edit current recipe</string>
   </property>
   <property name="statusTip">
    <string>Edit current recipe</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="action_collect_garbage">
   <property name="icon">
    <iconset resource="anymeal.qrc">
     <normaloff>:/images/trash.svg</normaloff>:/images/trash.svg</iconset>
   </property>
   <property name="text">
    <string>Collect &amp;Garbage</string>
   </property>
   <property name="toolTip">
    <string>Delete unused categories and ingredients</string>
   </property>
   <property name="statusTip">
    <string>Delete unused categories and ingredients</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="action_add_to_category">
   <property name="icon">
    <iconset resource="anymeal.qrc">
     <normaloff>:/images/categoryadd.svg</normaloff>:/images/categoryadd.svg</iconset>
   </property>
   <property name="text">
    <string>Add &amp;to Category</string>
   </property>
   <property name="toolTip">
    <string>Add selected recipes to a category</string>
   </property>
   <property name="statusTip">
    <string>Add selected recipes to a category</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="action_remove_from_category">
   <property name="icon">
    <iconset resource="anymeal.qrc">
     <normaloff>:/images/categoryremove.svg</normaloff>:/images/categoryremove.svg</iconset>
   </property>
   <property name="text">
    <string>&amp;Remove from Category</string>
   </property>
   <property name="statusTip">
    <string>Remove from Category</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="action_open_converter">
   <property name="icon">
    <iconset resource="anymeal.qrc">
     <normaloff>:/images/calculator.svg</normaloff>:/images/calculator.svg</iconset>
   </property>
   <property name="text">
    <string>Open Co&amp;nverter Window</string>
   </property>
   <property name="statusTip">
    <string>Open Converter Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="action_deduplicate">
   <property name="icon">
    <iconset resource="anymeal.qrc">
     <normaloff>:/images/duplicate.svg</normaloff>:/images/duplicate.svg</iconset>
   </property>
   <property name="text">
    <string>Remove duplicates</string>
   </property>
   <property name="toolTip">
    <string>Remove duplicates in selected recipes</string>
   </property>
   <property name="statusTip">
    <string>Remove duplicates in selected recipes</string>
   </property>
  </action>
  <action name="action_new">
   <property name="icon">
    <iconset resource="anymeal.qrc">
     <normaloff>:/images/new.svg</normaloff>:/images/new.svg</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>New recipe</string>
   </property>
   <property name="statusTip">
    <string>New recipe</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>reset_button</tabstop>
  <tabstop>title_edit</tabstop>
  <tabstop>with_category_radio</tabstop>
  <tabstop>without_category_radio</tabstop>
  <tabstop>category_edit</tabstop>
  <tabstop>with_ingredient_radio</tabstop>
  <tabstop>without_ingredient_radio</tabstop>
  <tabstop>ingredient_edit</tabstop>
  <tabstop>filter_button</tabstop>
  <tabstop>titles_view</tabstop>
  <tabstop>recipe_browser</tabstop>
 </tabstops>
 <resources>
  <include location="anymeal.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
